dali/ft/filecopy.cpp

```diff
@@ -349,98 +349,115 @@ bool FileTransferThread::launchFtSlaveCmd()
     MemoryBuffer msg;
     msg.setEndian(__BIG_ENDIAN);
     SocketEndpoint connectEP;
-    if (sprayer.useFtSlave)
+    HANDLE localFtSlaveHandle = 0; // used only if ftslave is launched on this host
+    Owned<IException> exception;
```

Comment:
NB: most of the apparent changes below are actually moving the code into a try/catch block.
The actual change is passing the extra localFtSlaveHandle handle to spawnRemoteChild and at the bottom (line 453) waiting on the process.